packages/discord.js/src/util/Constants.js

```diff
@@ -96,16 +117,7 @@ exports.NonSystemMessageTypes = [
  * * {@link ChannelType.GuildStageVoice}
  * @typedef {ChannelType[]} TextBasedChannelTypes
  */
-exports.TextBasedChannelTypes = [
-  ChannelType.DM,
-  ChannelType.GuildText,
-  ChannelType.GuildAnnouncement,
-  ChannelType.AnnouncementThread,
-  ChannelType.PublicThread,
-  ChannelType.PrivateThread,
-  ChannelType.GuildVoice,
-  ChannelType.GuildStageVoice,
-];
+exports.TextBasedChannelTypes = [...exports.TextBasedChannelTypes, ChannelType.DM];
```

Comment:
Might've meant this, otherwise its recursive

```suggestion
exports.TextBasedChannelTypes = [...exports.GuildTextBasedChannelTypes, ChannelType.DM];
```

Reply:
That was very smart of me